asyncapi/twilio/service_types.bal

```diff
@@ -0,0 +1,24 @@
+public type CallStatusService service object {
```

Comment:
Shall we properly document the remote functions in both services?.

Reply:
Add  documentations